cgmanifests/generated/cgmanifest.json

```diff
@@ -164,7 +174,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "3acac70a551c321574732e5bfd67930244bb7151",
+          "commitHash": "fc645b7626ebf86530dbd82fbece74d457e7ae07",
           "repositoryUrl": "https://github.com/emscripten-core/emsdk.git"
```

Comment:
What's the reason for this diff and the one for onnx-tensorrt?

Reply:
I am not sure, this file is generated.

I guess it is because someone updated [cgmanifests/cgmanifest.json](https://github.com/microsoft/onnxruntime/blob/480c793125c67e3c873f8083287571bf04f42f52/cgmanifests/cgmanifest.json), but forgot to run [cgmanifests/generate_cgmanifest.py](https://github.com/microsoft/onnxruntime/blob/master/cgmanifests/generate_cgmanifest.py)